src/bundle/Resources/views/admin/content_type/draft_list.html.twig

```diff
@@ -0,0 +1,92 @@
+{% trans_default_domain 'content_type' %}
+
+{% import "@ezdesign/admin/content_type/macros.html.twig" as macros %}
+
+{% form_theme form_draft_content_types_delete '@ezdesign/form_fields.html.twig' %}
+
+{% if pager.nbResults %}
```

Comment:
How do we handle tables where there are no items? I don't think rendering an empty page without any information is good.

Reply:
@webhdx both tables (defined and drafts) are rendered on the same page, if there are no drafts there is no need to show table header.